includes/admin/upgrades/v3/class-data-migrator.php

```diff
@@ -1038,9 +1038,15 @@ public static function tax_rates( $data = null ) {
 			return;
 		}
 
-		$scope = isset( $data['global'] )
+		$scope;
+
+		if ( $data['country'] === __( 'Global Rate', 'easy-digital-dowloads' ) ) {
```

Comment:
We shouldn't be doing any logical comparisons against a translatable value. This should also be flipped so it's a yoda conditional.